### PR TITLE
Fix layers content type rule to not include stereotypes

### PR DESCRIPTION
**Pull request by deanmarcussen**
Fixes logic

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs [hunk 1]
```diff
@@ -31,10 +32,10 @@ public ContentTypeConditionEvaluatorDriver(IConditionOperatorResolver operatorRe
         public override Task<IDisplayResult> DisplayAsync(ContentItem contentItem, BuildDisplayContext context)
         {
             // Do not include Widgets or any display type other than detail.
-            if (context.DisplayType == "Detail" && context.Shape.TryGetProperty(nameof(ContentTypeSettings.Stereotype), out string stereotype))
+            if (String.Equals("Detail", context.DisplayType, StringComparison.OrdinalIgnoreCase) && !context.Shape.TryGetProperty(nameof(ContentTypeSettings.Stereotype), out string stereotype))
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs:35]: Here that's fine to still use `context.DisplayType == "Detail"`, here `String.Equals()` will do the same wih one more method call, this to check the comparison type, and to do the same if it is `OrdinalIgnoreCase`

**Comment by sebastienros** [comment]
Is it the Rules module that defined when a widget is visible, and not the Layers?

File: src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs (hunk 1 above)
Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs:35]: And if `out string stereotype` is not used, you coul just use the `_` discard char

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs]: Good idea @jtkech Done.

**Comment by deanmarcussen** [comment]
> Is it the Rules module that defined when a widget is visible, and not the Layers?

Yes, I kept most of the rules, in rules (this particular one is a bit more layers though).

Mostly because the existing javascript layer methods get used in a lot of places, workflows etc, so while orientated at layers, are a bit more universal than just layers. The url one for example, I have used all over the place with workflows.

Open to moving them back to layers, or as a feature somewhere?

**Pull request merged**